client/src/App.js

```diff
@@ -243,8 +247,14 @@ function DocumentFromRecipe({ document }) {
   return sections;
 }
 
-function Prose({ content }) {
-  return <div dangerouslySetInnerHTML={{ __html: content }} />;
+function Prose({ name, content }) {
+  const headingText = HEADINGS[name];
+  return (
+    <>
+      <h2>{headingText}</h2>
```

Comment:
I miss the IDs. First of all, I'd love to be able to perma link to `https://mdn2/Web/HTML/Element/video#usage` and that should work without any JS. 
Second of all, once we have an ID (which acts as a native anchor), it would be nice to add one of those cool on-hover effects like GitHub does when you preview a Markdown document. Perhaps Schalk can build that for us in pure SCSS later.